doc/theoretical_description_calibration.rst

```diff
@@ -0,0 +1,88 @@
+.. title:: Theoretical Description : contents
+
+.. _theoretical_description_calibration
+
+=======================
+Theoretical Description
+=======================
+
+
+One method for multi-class calibration has been implemented in MAPIE so far :
+Top-Label [1].
+
+The goal of binary calibration is to transform a score (typically the one returned by an ML model) that has no mathematical meaning into a
+probability. The algorithms that are used for this type of calibration can be interpreted as estimators of the confidence level. Hence,
+they are made of an independent and dependent variable.
+
+The figure below illustrates what we would expect as a result from a calibration procedure, with the scores predicted being closer to the
+true accuracy compared to the original output.
+
+.. image:: images/calibration_basic.png
+   :width: 300
+   :align: center
+
+
+Firstly, we introduce binary calibration, we denote the :math:`(X, y)` pair as the score and ground truth for the object. Hence, :math:`y`
```

Comment:
:math:`(X, y)` : X is not the score, h(X) is the score

Reply:
done.